docs/changes.rst

```diff
@@ -263,6 +263,13 @@ Bug fixes
   embedded objects. This could have caused low-level exceptions to be raised
   at the pywbem API.
 
+* Fixed that a `for`-loop over `CIMInstance` / `CIMInstanceName` objects
```

Comment:
change 'Fixed that' to 'Fixed problem that'

Reply:
DONE